app/javascript/src/components/Common/CreatePane.jsx

```diff
@@ -2,24 +2,28 @@ import React from "react";
 
 import { Pane, Typography } from "neetoui";
 
-import Form from "./Form";
-
-export default function EditNotePane({
-  fetchNotes,
+export default function NewPane({
+  // fetchNotes,
```

Comment:
Not sure why you've pushed a `comment` like this to a PR. What's the point of this comment?

Reply:
It was just to make an idea of how this component will work while building it. Will remove all of them.